BUILD

```diff
@@ -14,10 +14,10 @@ exports_files(["LICENSE"])
 # build configuration
 ################################################################################
 
+# TODO(yannic): Remove in 3.14.0.
```

Comment:
Should we just remove all these now since 3.14.0 will be the next release?

Reply:
Seems like a good idea... I'll do that separately, though, in case anyone is relying on them. That will at least give a couple of commits to find the problem and flip behavior with just a flag.